src/images/logo.svg

```diff
@@ -0,0 +1,4 @@
+<svg width="150" height="100" viewBox="0 0 150 100" fill="none" xmlns="http://www.w3.org/2000/svg">
+<circle cx="50" cy="50" r="50" fill="black" fill-opacity="0.5"/>
+<circle cx="100" cy="50" r="50" fill="black" fill-opacity="0.15"/>
+</svg>
```

Comment:
Per my other comment, we should no longer be hosting a logo file in our repos as the logos come from the shared logo configuration settings and the CDN urls.